images/server/Containerfile

```diff
@@ -11,13 +11,16 @@ RUN SAMBACC_DISTNAME=latest \
 
 FROM registry.fedoraproject.org/fedora:35
 ARG INSTALL_PACKAGES_FROM=default
+ARG SAMBA_VERSION_SUFFIX=""
 ARG SAMBA_SPECIFICS=daemon_cli_debug_output
 
 MAINTAINER John Mulligan <jmulligan@redhat.com>
 
 COPY smb.conf /etc/samba/smb.conf
 COPY install-packages.sh /usr/local/bin/install-packages.sh
-RUN /usr/local/bin/install-packages.sh "$INSTALL_PACKAGES_FROM"
+RUN /usr/local/bin/install-packages.sh \
+    "$INSTALL_PACKAGES_FROM" \
+    "${SAMBA_VERSION_SUFFIX}"
```

Comment:
Style: either use curly braces in both variables substitutions, or in none.

Reply:
fixed